glide.lock

```diff
@@ -1,8 +1,10 @@
-hash: efb390ce6569c895e553582bd010493901e3fa4733fb9037970003c870cb5df2
-updated: 2016-06-27T11:56:08.014343879-07:00
+hash: a9ade02839b3b4a568225bf30592fbe944f4d74f60ec08e054d4b42f19f93271
+updated: 2016-06-27T11:39:10.012207091-07:00
 imports:
 - name: github.com/deis/controller-sdk-go
-  version: 3b6502cecedc6ebd7b0f42a8c01c31e8292d1e0d
+  version: 629d23864538f209adba0ce95304dfb6e5aab292
+  repo: https://github.com/Joshua-Anderson/controller-sdk-go.git
```

Comment:
This PR depends on deis/controller-sdk-go#16. I didn't want to marge that PR until this one has been LGTMed. The procedure we have been using to deal with this is to temporary point the depency to the PR branch, then merge SDK PR and update the vendored dependency to point at the main repo again.